python/pyspark/mllib/feature.py

```diff
@@ -447,6 +454,17 @@ class Word2Vec(object):
     >>> syms = model.findSynonyms(vec, 2)
     >>> [s[0] for s in syms]
     [u'b', u'c']
+
+    >>> import os, tempfile
+    >>> path = tempfile.mkdtemp()
+    >>> model.save(sc, path)
+    >>> sameModel = Word2VecModel.load(sc, path)
+    >>> model.transform("a") == sameModel.transform("a")
```

Comment:
Sure, but I think it would be better to just write
`sameModel.transform(a)` separately and the output.

However, I have no objections either !